eng/targets/Helix.Common.props

```diff
@@ -27,8 +27,8 @@
 
   <!-- daily scheduled only queues -->
   <ItemGroup Condition="'$(IsWindowsOnlyTest)' != 'true' AND '$(TargetArchitecture)' == 'x64' AND '$(IsHelixDaily)' == 'true'">
+    <HelixAvailableTargetQueue Include="Windows.81.Amd64.Open" Platform="Windows" />
     <HelixAvailableTargetQueue Include="Ubuntu.1804.Amd64.Open" Platform="Linux" />
-    <HelixAvailableTargetQueue Include="OSX.1014.Amd64.Open" Platform="Linux" />
```

Comment:
Just curious: Do we have a `macOS` or `OSX` platform? Seems less like our Linux platforms every day 😈

Reply:
Platform is really windows or not windows at this point :)